tests/driver_bme680/main.c

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (C) 2018 Mesotic SAS
+ *               2020 Gunar Schorcht
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Test application for the bme680_driver package.
+ *
+ * @author      Dylan Laduranty <dylan.laduranty@mesotic.com>
+ * @author      Gunar Schorcht <gunar@schorcht.net>
+ * @}
+ */
+
+#include <stdio.h>
+
+#include "board.h"
+#include "bme680.h"
+#include "bme680_params.h"
+#include "mutex.h"
+#include "xtimer.h"
+
+#define BME680_TEST_PERIOD_US   (5 * US_PER_SEC)
+
+xtimer_t timer;
+
+static void _timer_cb(void *arg)
+{
+    xtimer_set(&timer, BME680_TEST_PERIOD_US);
+    mutex_unlock(arg);
+}
+
+int main(void)
+{
+    mutex_t timer_mtx = MUTEX_INIT_LOCKED;
+
+    bme680_t dev[BME680_NUMOF]; // [BME680_NUMOF];
```

Comment:
Please change the C++ style comment

Reply:
Oh, yes, it's just an accidental leftover.